README.md

```diff
@@ -1,2 +1,19 @@
-# hipo-typescript-eslint-config
-Hipo's ESLint rules for TypeScript
+# @hipo/eslint-config-typescript
+
+Hipo's shareable ESLint configuations for TypeScript.
+
+## Installation
+
+```
+npm i --save-dev @hipo/eslint-config-typescript
+```
+
+## Usage
+
+Set your eslint config to:
```

Comment:
```suggestion
Update your ESLint config file (`.eslintrc.*`) to use the `"@hipo/eslint-config-typescript"` by adding it inside extended configuration array:
```
I think we can improve description like this

Reply:
Thank you for your comments!